src/routes/login/+page.svelte

```diff
@@ -1 +1,98 @@
-<h1>Login</h1>
+<script>
```

Comment:
Can you please try to use TypeScript instead?
```suggestion
<script lang="ts">
```